<launch>
	<node pkg="vis"
		type="lineNode"
		name="vis_left"/>
	<param name="Debug_Flag" type="bool" value="true" />
	
	<param name="Camera_Source_Topic" type="string" value="/usb_cam2/image_raw" />
	<param name="Output_Image_topic" type="string" value="/output_video_left" />
	<param name="Output_Pointlist_topic" type ="string" value="/output_point_list_left"/>
	<param name="PointListOut" type="bool" value="true" />
	
	<param name="Blob_Size" type="int" value="20" />
	<param name="Fixed_Threshold" type="bool" value="false" />
	<param name="Adaptive_HSV_Max_S" type ="int" value = "64"/>
	<param name="Adaptive_HSV_Min_V" type ="int" value = "-24"/>
	<param name="Adaptive_HSV_MPatch_Size" type ="int" value = "25"/>
	
	<param name="OpenCVMatFile" type="string" value="$(find vis)/launch/vision_params_left.yaml" />
	<!-- FYI: OpenCV Image coordinates are in [x,y], where X is a column index (0 left), and Y is a row index (0 top) -->
</launch>
